urth_dash_js/notebook/deploy-menu/deploy-menu.js

```diff
@@ -80,5 +80,25 @@ define([
             .on('click', function() {
                 do_bundle('zip');
             });
+        $li = $('<li>')
+        .insertAfter($('#download_ipynb'));
+        $('<a>')
```

Comment:
Add href set attr here like in the others. Fixes incorrect cursor set by browser.